vpdq/python/tests/test_vpdq_hash.py

```diff
@@ -28,39 +28,53 @@
     "pattern-sd-with-small-logo-bar",
```

Comment:
Why are these changes included in this diff? Isn't it fixing the str?

Reply:
Reverting those!